### ISSUE TITLE
bigip_node unable to select none for monitoring type

### ISSUE THREAD

**Issue by twittyc** [F5Networks/f5-ansible#804]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
bigip_node

##### ANSIBLE VERSION
```
ansible 2.5.3
  config file = None
  configured module search path = ['/Users/ct046686/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.6/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.6.5 (default, Apr 25 2018, 14:23:58) [GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.1)]
```

##### PYTHON VERSION
```
Python 2.7.15
```

##### BIGIP VERSION

```
BIG-IP 12.1.3.1 Build 0.39.9 Engineering Hotfix
```

##### LIBRARY VERSIONS

```
f5-sdk==3.0.15
```

##### SUMMARY
Unable to set node monitor type to none when using the bigip_node module. If this option is available it is not in the Ansible documentation.

**Comment by caphrim007** [comment]
@twittyc can you provide a task that reproduces this error?

**Comment by caphrim007** [comment]
@twittyc so you just want to remove the monitors from the node?

**Comment by twittyc** [comment]
Correct. I do want a monitor on these nodes. However, if I leave out the monitors property it will default to icmp.

**Comment by mitchellmaler** [comment]
Any update on this? I am having the same issue. We are not allowed to enable icmp on the networks our nodes are on and I already have pool monitoring so I would like to set node monitoring to none.

**Comment by hatchepsut** [comment]
I have a small fix that seem to work with version 2.6.3. Not sure how to go forward though.  Can someone give me a pointer?

**Comment by caphrim007** [comment]
@hatchepsut submit PR's to this repository here.
